lib/node_modules/@stdlib/math/base/special/cidentity/examples/c/example.c

```diff
@@ -17,19 +17,31 @@
 */
 
 #include "stdlib/math/base/special/cidentity.h"
+#include "stdlib/complex/float64.h"
+#include "stdlib/complex/reim.h"
 #include <stdio.h>
-#include <complex.h>
 
 int main( void ) {
```

Comment:
@kgryte, I'm not sure if we want to leave the `void` here. Can you please confirm this before merging?

Reply:
@steff456 We do. In C, this is how one communicates that a function accepts zero-arguments. We had previously been inconsistent with including `void` (e.g., examples tended not to include, but benchmarks did include). Not including it is deprecated behavior. Hence, its current inclusion and best practice going forward.

Reply:
Great! I'll take it into account for the open PRs and the ones coming 😊